lib/react/server_rendering/exec_js_renderer.rb

```diff
@@ -10,7 +10,9 @@ class ExecJSRenderer
 
       def initialize(options={})
         js_code = options[:code] || raise('Pass `code:` option to instantiate a JS context!')
-        @context = ExecJS.compile(GLOBAL_WRAPPER + js_code)
+        full_code = GLOBAL_WRAPPER + js_code
+        # File.write("./test/dummy/tmp/latest_js_context.js", full_code)
```

Comment:
@Judahmeek keep this comment?

Reply:
I had to use it to debug two different SSR error with ExecJS so yes, I'd like to keep it.